app/src/main/res/layout/activity_system_search.xml

```diff
@@ -107,19 +107,20 @@
     </com.google.android.material.appbar.AppBarLayout>
 
     <androidx.core.widget.NestedScrollView
+        android:id="@+id/results"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:clipToPadding="false"
-        android:elevation="4dp"
+        android:elevation="@dimen/systemSearchResultsElevation"
         android:paddingBottom="4dp"
-        android:id="@+id/results"
         app:layout_behavior="@string/appbar_scrolling_view_behavior">
 
         <androidx.constraintlayout.widget.ConstraintLayout
+            android:id="@+id/resultsContent"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
             android:background="?attr/colorPrimary"
-            android:backgroundTint="?attr/colorPrimary"
+            android:clipToPadding="false"
             android:elevation="4dp">
```

Comment:
Just seen that parent has also 4dp elevation as `systemSearchResultsElevation`. Should this be using the same value or remove the property?

Reply:
Similar to my last comment the elevation values happen to be the same but shouldn't be tightly coupled. The reason `systemSearchResultsElevation` exists is because we need it in code as we set slightly different shadows when the onboarding is shown, otherwise I would delete it. Let me know if you want to zoom to go over this in more detail. I'll proceed with the PR but can create a follow-up one if any changes come from a discussion.